exercises/follow_line/web-template/assets/ws_code.js

```diff
@@ -53,12 +72,32 @@ function stopCode(){
     websocket_code.send(stop_code);
 }
 
+// Function to save the code locally
+function saveCodeLocally(){
+	var python_code = editor.getValue();
+	var blob = new Blob([python_code], {type: "text/plain; charset=utf-8"});
+	if (window.navigator.msSaveOROpenBlob)
+		window.navigator.msSaveOrOpenBlob(blob, "follow_line.py");
+	else{
+		var a = document.createElement("a"),
+		url = URL.createObjectURL(blob);
+		a.href = url;
+		a.download = "follow_line.py";
+		document.body.appendChild(a);
+		a.click()
+		setTimeout(function(){
+			document.body.removeChild(a);
+			window.URL.revokeObjectURL(url);
+		}, 0);
+	}
+}
+
 // Function to save the code
 function saveCode(){
```

Comment:
I think we can comment the code that uses websocket connection to save and load.